configure

```diff
@@ -1440,6 +1446,10 @@ Optional Packages:
   --with-turbocap[=DIR]   include Riverbed TurboCap support (located in
                           directory DIR, if supplied). [default=yes, if
                           present]
+  --with-dpdk-includes=IDIR
```

Comment:
Will the DPDK include and library directories ever be anything other than the `include` and `lib` subdirectories of some particular higher-level directory, such as `/usr/local/include` and `/usr/local/lib`, or `/opt/dpdk/include` and `/opt/dpdk/lib`, or something such as that?

Reply:
1. You could find the system requirement about DPDK. https://doc.dpdk.org/guides/linux_gsg/sys_reqs.html

Reply:
Then why not just handle DPDK the same way other optional modules are handled, with a `--with-dpdk` option that takes an optional "DIR" argument, with the header directory being `{DIR}/include` and the library directory being `{DIR}/lib`, so that:

- if you don't specify `--with-dpdk`, it tries to find DPDK in `/usr` and `/usr/local` and, if it finds them, builds with DPDK, otherwise doesn't build with DPDK;
- if you specify `--with-dpdk`, it tries to find DPDK in `/usr` and `/usr/local` and, if it finds them, builds with DPDK, otherwise fails;
- if you specify `--with-dpdk=DIR`, it tries to find DPDK in DIR and, if it finds them, builds with DPDK, otherwise fails;
- if you specify `--without-dpdk`, it doesn't try to find DPDK and doesn't build with DPDK?

For CMake, have a `cmake/Modules/FindDPDK.cmake` module.

Reply:
OK. I will try to do that on the weekend.

Reply:
I rewrite the configuration files with your suggestion. Thanks.